recipe/meta.yaml

```diff
@@ -162,7 +164,7 @@ outputs:
         - async-timeout
         - colorful
         - gpustat >=1.0.0
-        - grpcio
+        - grpcio {{ grpcio_version }}
```

Comment:
@hmaarrfk what is the point of these pins? All outputs depend on the identical version of ray-core, which now exports a runtime dependency on a compatible libgrpc. I don't think this extra grpcio_version thing is required.

Reply:
ah i see you are right. indeed these pins are not necessary. sorry for the noise.

Reply:
Please revert

Reply:
Done